packages/sources/css/src/components/button/src/index.css

```diff
@@ -89,42 +89,59 @@
 
 .vtmn-btn_variant--primary-reversed:not([disabled]):hover {
   background-color: color-mod(var(--vtmn-color_white) alpha(80%));
+  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_white);
 }
 
 .vtmn-btn_variant--primary-reversed:not([disabled]):active {
   background-color: color-mod(var(--vtmn-color_white) alpha(60%));
+  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_white);
 }
 
 /* Secondary button */
 .vtmn-btn_variant--secondary {
-  background-color: transparent;
-  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_black);
-  color: var(--vtmn-color_black);
-  fill: var(--vtmn-color_black);
+  background-color: var(--vtmn-color_white);
+  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_grey-light-2);
+  color: var(--vtmn-color_brand-digital-dark-1);
+  fill: var(--vtmn-color_brand-digital-dark-1);
 }
 
 .vtmn-btn_variant--secondary:not([disabled]):hover {
-  background-color: color-mod(var(--vtmn-color_grey-dark-2) alpha(5%));
+  background-color: var(--vtmn-color_blue-light-1);
 }
 
 .vtmn-btn_variant--secondary:not([disabled]):active {
-  background-color: color-mod(var(--vtmn-color_grey-dark-2) alpha(15%));
+  background-color: var(--vtmn-color_blue-light-2);
 }
 
-/* Secondary reversed button */
-.vtmn-btn_variant--secondary-reversed {
-  background-color: transparent;
-  box-shadow: inset 0 0 0 rem(2px) var(--vtmn-color_white);
-  color: var(--vtmn-color_white);
-  fill: var(--vtmn-color_white);
+/* Tertiary button */
+.vtmn-btn_variant--tertiary {
+  background-color: var(--vtmn-color_brand-digital-light-3);
+  color: var(--vtmn-color_brand-digital-dark-1);
+  fill: var(--vtmn-color_brand-digital-dark-1);
+}
+
+.vtmn-btn_variant--tertiary:not([disabled]):hover {
+  background-color: var(--vtmn-color_blue-light-3);
+}
+
+.vtmn-btn_variant--tertiary:not([disabled]):active {
+  background-color: var(--vtmn-color_blue-light-4);
+}
+
+/* Conversion button */
+.vtmn-btn_variant--conversion {
+  box-shadow: inset 0 0 0 rem(2px) transparent;
+  background-color: var(--vtmn-color_yellow);
+  color: var(--vtmn-color_black);
+  fill: var(--vtmn-color_black);
 }
 
-.vtmn-btn_variant--secondary-reversed:not([disabled]):hover {
-  background-color: color-mod(var(--vtmn-color_white) alpha(20%));
+.vtmn-btn_variant--conversion:not([disabled]):hover {
```

Comment:
Breaking change, we cannot remove "secondary reversed" CSS class without making a major version. So for the moment, you can move secondary-reversed classes to the bottom of this file under a "Deprecated" comment.

Reply:
Done